### PR TITLE
🐛 Delayed force-exit `gulp test` after Karma reports test completion

### PR DESCRIPTION
**Pull request by rsimha**
There are several potential causes for the disconnect / timeout issues we're seeing on Travis, where Karma reports that all tests are complete, but `gulp test` doesn't terminate because something is holding it up from doing so.

For example, see https://travis-ci.org/ampproject/amphtml/jobs/370168309

**Related issues:**
- https://github.com/karma-runner/karma/issues/2980
- https://github.com/karma-runner/karma/issues/1035
- https://github.com/karma-runner/karma/issues/1788
- https://github.com/karma-runner/karma-sauce-launcher/issues/95

We have a ticket open with Sauce labs to investigate this.

Meanwhile, this PR temporarily mitigates the problem by force-exiting the `gulp test` process after a gap of 5 seconds, so that Karma reporters can print all their output to the console.

### CODE REVIEW

**Comment by rsimha** [comment]
Tested this locally, and I can verify that `gulp test` now exits after all tests are complete.

File: build-system/tasks/runtime-test.js [hunk 1]
```diff
@@ -371,7 +371,10 @@ function runTests() {
       log(
           red('ERROR:'),
           yellow('Karma test failed with exit code ' + exitCode));
-      process.exitCode = exitCode;
+      // TODO(rsimha): Remove this after Karma / Sauce ticket is resolved.
```
Comment (cramforce) [inline build-system/tasks/runtime-test.js]: Should probably link to issues here.

Comment (rsimha) [reply build-system/tasks/runtime-test.js]: Done.

**Pull request merged**